src/main/java/uk/gov/hmcts/reform/sscs/extractor/DwpResponseReceivedDateExtractor.java

```diff
@@ -23,8 +33,8 @@
                 return Optional.of(event.getValue().getDateTime());
             }
         }
-
-        return Optional.ofNullable(caseData.getDwpResponseDate()).map(date -> ZonedDateTime.parse(date + "T00:00:00Z").toInstant().atZone(ZoneId.of(ZONE_ID)));
+        LocalTime time = LocalTime.MIN.plusSeconds(rand.nextInt(60 * 60 * 1000));
```

Comment:
Is * 1000 (for ms?) here correct given you are using plus seconds?

Reply:
its just number, random will pick the value and bound to 24hours

Reply:
Should it not be 60 * 60 * 24 in that case?

60*60*24 = 86,400 = seconds in a day
60*60*1000 = 3,600,000.

For 60*60*1000 76% of the rands generated will be above 86,400 seconds in the day and therefore all schedule at 23:59